src/components/VoiceSearch.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div
+    v-if="state"
+    :class="suit()"
+  >
+    <slot
+      :isBrowserSupported="state.isBrowserSupported"
```

Comment:
```suggestion
      :is-browser-supported="state.isBrowserSupported"
```

idem on the other ones